src/virtualenv.py

```diff
@@ -1542,6 +1542,10 @@ def install_activate(home_dir, bin_dir, prompt=None):
 
     files["activate_this.py"] = ACTIVATE_THIS
 
+    if majver >= 3:
```

Comment:
you should minor version qualify here the same as you do with the install before anyone with Python 3.3 turns up that it's not working

Reply:
done!